wrappers/node/test/integration/ledger.test.ts

```diff
@@ -20,13 +20,13 @@ describe('wallet', () => {
     expect(pwInfo.pw_vk.length).equal(44);
   });
 
-  it('write new endorser did from seed', async () => {
-    const genesisPath = path.join(__dirname, '/../../resources/localhost.txn');
-    await openMainPool({ genesis_path: genesisPath });
-    const pwInfo = await createAndStoreDid(seed);
-    expect(pwInfo.pw_did).equal('FhrSrYtQcw3p9xwf7NYemf');
-    expect(pwInfo.pw_vk).equal('91qMFrZjXDoi2Vc8Mm14Ys112tEZdDegBZZoembFEATE');
-  });
+  // it('write new endorser did from seed', async () => {
```

Comment:
this should be restorable?

Reply:
Idk what was up with it, but it did not work. If you consider it important, I can look into restoring it. That was actually the whole point of this, to kinda cherry pick the things worth putting the effort into restoring and delete those mock-only duplicate libvcx/node wrapper tests.

So, should I look into this?

Reply:
Nvm, restored. The seed was bad in the test, but the mocks were hiding that because they weren't doing anything 🙃